src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -29,6 +29,43 @@ public new abstract class Arm64 : AdvSimd.Arm64
         }
 
 
+
+        ///  Count16BitElements : Count the number of 16-bit elements in a vector
+
+        /// <summary>
+        /// uint64_t svcnth_pat(enum svpattern pattern)
+        ///   CNTH Xresult, pattern
+        /// </summary>
+        public static unsafe ulong Count16BitElements([ConstantExpected] SveMaskPattern pattern = SveMaskPattern.All) => Count16BitElements(pattern);
```

Comment:
`cnt*` also has `imm` that defaults to `1`. How do we let user pass it to the API?

https://docsmirror.github.io/A64/2023-06/cntb_r_s.html

Reply:
The `imm` is just a multiplier for the result (eg: count elements in 4 vectors). It was decided at review not to have this in the API.
In the future we could add a lowering mechanism to check for multiply and cnt. But that'll need to be done in the multiply lowering - you would be looking for `int x = 4 * Count16BitElements(all)`. I can open a ticket for that?